modules/internal/OwnedObject.chpl

```diff
@@ -156,6 +156,11 @@ module OwnedObject {
       of the argument. The result has the same type as the argument.
       If the argument is non-nilable, it must be recognized by the compiler
       as an expiring value.
+
+      .. note::
+
+         This is part of an experimental interface, users should prefer
+         :proc:`owned.create`/:proc:`owned.retain`/:proc:`owned.clear` for now.
```

Comment:
I'd phrase this differently, since we do intend to replace create, retain and clear with them.

Maybe something like:
```
This is part of a new interface to replace <specific one(s) this method is replacing>.  However,
it has not been used as widely as <old method(s)> yet, so there may be some bugs we haven't found.
If you discover any bugs with it, please report them to us and fall back on <old method(s)> in the
meanwhile.
```